Webapp/sources/services/novel_compound.py

```diff
@@ -192,15 +192,13 @@ def extract_compound_name(primary_key: str) -> Optional[str]:
     name_match = re.search(pattern_single, primary_key)
     if name_match:
         compound_name = name_match.group(1)
-        unescaped_compound_name = bytes(compound_name, "utf-8").decode("unicode_escape")
-        return unescaped_compound_name
+        return compound_name
 
     # Try matching with double quotes pattern if single quotes pattern fails
     name_match = re.search(pattern_double, primary_key)
     if name_match:
         compound_name = name_match.group(1)
-        unescaped_compound_name = bytes(compound_name, "utf-8").decode("unicode_escape")
-        return unescaped_compound_name
+        return compound_name
```

Comment:
For this bit you can replace the two regexes with just the one: `([\"\'])(.+?)\1,\s*\d+`. Paste it into the following link and select Python on the left hand side to check: https://regex101.com/